src/Cake.Core/Tooling/ToolResolutionStrategy.cs

```diff
@@ -122,6 +127,7 @@ private FilePath LookInPath(string tool)
                     {
                         if (_fileSystem.Exist(file))
                         {
+                            _log.Verbose($"Resolved tool to path {file}");
```

Comment:
I would prefer this to be diagnostic.

Reply:
Oh - I looked for a Diagnostic method but couldn't find one. Will look again.

Reply:
Sorry for that! 😞 
Diagnostic is the verbosity. Debug will do fine since it uses the diagnostic verbosity.